src/stores/business.ts

```diff
@@ -67,9 +67,10 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
   }
 
   /** Return the business details for the given identifier */
-  async function getBusinessDetails (identifier: string, params?: object) {
+  async function getBusinessDetails (identifier: string, params?: object, slim: boolean = false) {
+    const url = `${apiURL}/businesses/${identifier}${slim ? '?slim=true' : ''}`
```

Comment:
Is this function the one use to load all business data? Won't it break other functionality if you're now fetch the slim results instead of full results?

Reply:
Answer: see `slim` parameter.